modules/manager/controller_setup.go

```diff
@@ -386,6 +388,15 @@ func SetupControllers(mgr manager.Manager, c *Config) (map[string]ControllerDef,
 					konnect.WithKonnectEntitySyncPeriod[configurationv1alpha1.KongUpstream](c.KonnectSyncPeriod),
 				),
 			},
+			KongCACertificateControllerName: {
+				Enabled: c.KonnectControllersEnabled,
+				Controller: konnect.NewKonnectEntityReconciler[configurationv1alpha1.KongCACertificate](
```

Comment:
This type tag is deduced (see above in other calls to `NewKonnectEntityReconciler`)

```suggestion
				Controller: konnect.NewKonnectEntityReconciler(
```